src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searchBar {
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+  width: 100%;
+  margin: 20px 0;
+  height: 70px;
+  padding: 25px 0 25px 62px;
+}
+
+.searchBar--small {
+  height: 42px;
+  padding: 10px 0 11px 33px;
+}
+
+.searchBar::placeholder {
```

Comment:
try here font-family: inherit; as well, this may fix test

Reply:
> try here font-family: inherit; as well, this may fix test

 I have tried it with the same result, one test was failed ((